docs/code/BSTLD.py

```diff
@@ -4,13 +4,8 @@
 #
 
 # pylint: disable=wrong-import-position
-# pylint: disable=wrong-import-order
```

Comment:
about the commit message,
should it be `ci: delete useless disable comments of pylint`